torch/csrc/distributed/rpc/rpc_command_base.h

```diff
@@ -1,7 +1,9 @@
 #pragma once
 
 #include <torch/csrc/distributed/rpc/message.h>
+#include <torch/csrc/distributed/rpc/rpc_agent.h>
 #include <torch/csrc/distributed/rpc/types.h>
+#include <torch/csrc/jit/serialization/pickle.h>
```

Comment:
I saw these two headers are included in the .cpp files in the derived classes. Do we still need them here? Will this create circular dependency?